test/unit/mwc-button.test.js

```diff
@@ -35,17 +37,29 @@ test('initializes as an mwc-button', () => {
   assert.instanceOf(element, Button);
 });
 
-suite('mwc-fab');
+test('get/set disabled updates the disabled property on the native button element', async () => {
+  element.disabled = true;
+  await afterNextRender();
+  const button = element.shadowRoot.querySelector('button');
+  assert.equal(button.hasAttribute('disabled'), true);
 
-beforeEach(() => {
-  element = document.createElement('mwc-fab');
-  document.body.appendChild(element);
+  element.disabled = false;
+  await afterNextRender();
```

Comment:
So, LitElement provides a `renderComplete` Promise that we should use for this. 

NOTE: It's a little tricky because `ComponentElement` provides `componentReady` (as a method) and probably should instead override `renderComplete`. Button just extends LitElement so that's not an issue here.

Reply:
Thanks for pointing that out, seems much cleaner to have this signal explicitly come from the element.